2020-01-Winter/3_stacks_queues/recent_counter/recent_counter_java.java

```diff
@@ -0,0 +1,46 @@
+package com.company;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class Main {
```

Comment:
This file doesn't compile as is :(
I had to split the classes into two files and remove the package name to get it to compile and run as expected